CMakeLists.txt

```diff
@@ -53,16 +68,16 @@ add_subdirectory(dataformats)
 
 # Should unit_test_targets depend on default targets ???
 
+add_custom_target(unit_tests
+  DEPENDS ${unit_test_targets})
 add_custom_target(runtest
     COMMAND ${CMAKE_CTEST_COMMAND} -V -R regular_*
-    DEPENDS ${unit_test_targets})
+    DEPENDS ${unit_tests})
```

Comment:
The target `runtest` should depend on the target `unit_tests` and not on `${unit_tests}`.